docs/paper/dev/api/command-api/arguments.mdx

```diff
@@ -0,0 +1,186 @@
+| PARTICLE_TYPE       | Particle              | [Every particle type](https://minecraft.wiki/w/Particles_(Java_Edition)#Types_of_particles)                                                                                                                                                                                                                     |
+| POTION              | PotionType            | [Every potion type](https://minecraft.wiki/w/Potion#Effect_potions)                                                                                                                                                                                                                          |
+| SOUND_EVENT         | Sound                 | [Events that trigger sound effects](https://minecraft.wiki/w/Sounds.json#Sound_events)                                                                                                                                                                                                       |
+| VILLAGER_PROFESSION | Villager.Profession   | [Villager professions](https://minecraft.wiki/w/Villager#Professions)                                                                                                                                                                                                                        |
+| VILLAGER_TYPE       | Villager.Type         | [Villager biome specific type](https://minecraft.wiki/w/Villager#Professions)                                                                                                                                                                                                                |
+| MEMORY_MODULE_TYPE  | MemoryKey             | Keys for saving per-entity data                                                                                                                                                                                                                                                              |
+| FROG_VARIANT        | Frog.Variant          | [Frog variants](https://minecraft.wiki/w/Frog)                                                                                                                                                                                                                                               |
+| MAP_DECORATION_TYPE | MapCursor.Type        | [Types of sprites displayed on a map](https://minecraft.wiki/w/Map#Map_icons)                                                                                                                                                                                                                |
+| FLUID               | Fluid                 | [Fluid types](https://minecraft.wiki/w/Fluid)     
+
+Brigadier itself also specifies many argument types. For more information on them, see <Javadoc name={"io.papermc.paper.command.brigadier.argument.ArgumentTypes"}>ArgumentTypes</Javadoc>
```

Comment:
```suggestion
Minecraft itself also specifies many argument types. For more information on them, see <Javadoc name={"io.papermc.paper.command.brigadier.argument.ArgumentTypes"}>ArgumentTypes</Javadoc>
```
I think it makes more sense this way since Brigadier itself only adds [these](https://github.com/Mojang/brigadier/tree/master/src/main/java/com/mojang/brigadier/arguments).
The arguments provided in the `ArgumentTypes` class are vanilla argument types.

Reply:
No, Brigadier is the command API in Minecraft, so it's just more specific to use Brigadier over Minecraft

Reply:
Yeah, Brigadier is the command framework Minecraft uses, but it doesn't provide the argument types that `ArgumentTypes` gives access to.
If you use plain Brigadier, you don't have access to those arguments but only to argument types that represent basic Java types.